File: src/scanner/code/__fixtures__/python/comments.py
"""Test fixture: Crypto mentions only in comments and strings — no real crypto calls."""

# rsa.generate_private_key(public_exponent=65537, key_size=2048)
# hashlib.md5(b"data")

"""
This is a docstring that mentions RSA and MD5:
rsa.generate_private_key(public_exponent=65537, key_size=2048)
hashlib.md5(b"data")
"""

message = "Use rsa.generate_private_key() for RSA keys"
another = 'hashlib.md5(b"test") is insecure'

def no_crypto():
    """This function mentions ec.generate_private_key(ec.SECP256R1()) but does not call it."""
    pass

File: src/scanner/code/__fixtures__/javascript/comments.js
/**
 * Test fixture: Crypto mentions only in comments — no real calls.
 * crypto.generateKeyPairSync('rsa', { modulusLength: 2048 })
 * crypto.createHash('md5')
 */

// crypto.generateKeyPairSync('rsa', { modulusLength: 2048 })
// crypto.createHash('md5')

const message = "Use crypto.generateKeyPairSync('rsa') for RSA keys";
const another = "crypto.createHash('md5') is insecure";

function noCrypto() {
  return "hello";
}
